components/runtime/node8/Makefile

```diff
@@ -0,0 +1,58 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2015 EveryCity Ltd. All rights reserved.
+# Copyright 2016 Alexander Pyhalov
+# Copyright 2016, 2017 Andreas Wacknitz
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		node
+COMPONENT_FMRI=		runtime/nodejs-8
+COMPONENT_SUMMARY=	Evented I/O for V8 JavaScript.
+COMPONENT_CLASSIFICATION=	Development/Other Languages
+COMPONENT_PROJECT_URL=	http://nodejs.org
+COMPONENT_VERSION=	8.0.0
+COMPONENT_LICENSE=	BSD-like, MIT, Apache
+COMPONENT_LICENSE_FILE=	node.license	
+COMPONENT_SRC=		$(COMPONENT_NAME)-v$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.xz
+COMPONENT_ARCHIVE_HASH=	sha256:258d3aa3632bbbf2a447c5d524bd7074e23aa26e2b698774ff452954f6174f1c
+COMPONENT_ARCHIVE_URL=	http://nodejs.org/dist/v$(COMPONENT_VERSION)/$(COMPONENT_ARCHIVE)
+
+include $(WS_MAKE_RULES)/prep.mk
+include $(WS_MAKE_RULES)/configure.mk
+include $(WS_MAKE_RULES)/ips.mk
+
+GCC_VERSION=7
+
+# Include /usr/sbin for dtrace
+PATH=/usr/bin:/usr/gnu/bin:/usr/sbin
+
+CONFIG_SHELL =	$(PYTHON)
+
+COMPONENT_PRE_CONFIGURE_ACTION = ($(CLONEY) $(SOURCE_DIR) $(@D))
+
+CONFIGURE_SCRIPT =	$(@D)/configure
+
+CONFIGURE_OPTIONS= --prefix=/usr/node/8
+
+build:		$(BUILD_32)
+
+install:	$(INSTALL_32)
+
+test:       $(NO_TESTS)
```

Comment:
There are tests but they haven't been used in the past (node, node6, node7) and I only run them manually yet. There is still some work left to use them.
Maybe in a later PR?